.github/workflows/build.yml

```diff
@@ -12,7 +12,7 @@ jobs:
           - stable
           - beta
           - nightly
-          - 1.61.0 # our MSRV
+          - 1.64.0 # our MSRV
```

Comment:
At this point, why not take it to 1.71 so it'll take longer until we have to pay attention to this again?